tests/e2e/configurer/upgrade.go

```diff
@@ -140,6 +142,23 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 	// test lock and add to existing lock for both regular and superfluid lockups (only chainA)
 	chainA.LockAndAddToExistingLock(sdk.NewInt(1000000000000000000), poolShareDenom, lockupWalletAddrA, lockupWalletSuperfluidAddrA)
 
+	// LP to pools 833, 817, 810
+	// initialize lp wallets
+	lpWalletAddr := chainANode.CreateWallet(lpWallet)
+	// JoinPool 833 from lpWallet
+	// TODO: updaet the amounts based on the pool initialization
```

Comment:
This is great! Let's wait until https://github.com/osmosis-labs/osmosis/pull/4393 is in, then we can finalize this